README.md

```diff
@@ -12,85 +12,141 @@ providers, GCE has a couple of advantages for Chef cookbook testing:
 
 ## Requirements
 
-Ruby 1.9 or greater.
+### Ruby Version
 
+Ruby 2.0 or greater.
+
+### Google Cloud Platform (GCP) Project
 A [Google Cloud Platform](https://cloud.google.com) account is
 required.  If you do not already have an appropriate "project" in
 which to run your test-kitchen instances, create one, noting the
-"project id".  Then, within the [Google API
-Console](https://code.google.com/apis/console/), create a "service
-account" for the project under the "API Access" tab.  Save the key
-file, and note the email address associated with the service account
-(e.g. 123456789012@developer.gserviceaccount.com - not the project
-owner's email address).
-
-If you are not using the `public_key_path` setting (see below) and
-have not [set up SSH keys for your GCE
-environment](https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys),
-you must also do that prior to using kitchen-gce.  Also, you will
-likely want to add your SSH keys to ssh-agent prior to converging any
-instances.
+"project id".
+
+### Authentication and Authorization
+
+The [underlying API](https://github.com/google/google-api-ruby-client) this plugin uses relies on the 
+[Google Auth Library](https://github.com/google/google-auth-library-ruby) to handle authentication to the
+Google Cloud API. The auth library expects that there is a JSON credentials file located at:
+
+`~/.config/gcloud/application_default_credentials.json`
+
+The easiest way to create this is to download and install the [Google Cloud SDK](https://cloud.google.com/sdk/) and run the
+`gcloud auth login` command which will create the credentials file for you.
+
+If you already have a file you'd like to use that is in a different location, set the
+`GOOGLE_APPLICATION_CREDENTIALS` environment variable with the full path to that file.
+
+### SSH Keys
+
+In order to bootstrap Linux nodes, you will first need to ensure your SSH
+keys are set up correctly. Ensure your SSH public key is properly entered 
+into your project's Metadata tab in the GCP Console. GCE will add your key
+to the appropriate user's `~/.ssh/authorized_keys` file when Chef first
+connects to perform the bootstrap process.
+
+ * If you don't have one, create a key using `ssh-keygen`
+ * Log in to the GCP console, select your project, go to Compute Engine, and go to the Metadata tab.
+ * Select the "SSH Keys" tab.
+ * Add a new item, and paste in your public key.
+    * Note: to change the username automatically detected for the key, prefix your key with the username
+      you plan to use to connect to a new instance. For example, if you plan to connect
+      as "chefuser", your key should look like: `chefuser:ssh-rsa AAAAB3N...`
+ * Click "Save".
+
```

Comment:
The only problem I have with this method is that Test Kitchen / net-ssh doesn't consider `~/.ssh/google_compute_engine` to be a default key, so every user that follows these instructions also needs to edit every .kitchen.yml they use to ensure that key is passed in. One of the largest sources of frustration for many TK users is having authentication fail but not know why.

I think it's good to give users many options though so I'll add this in as an alternative.